src/pages/signin/SignInPage.tsx

```diff
@@ -200,6 +217,11 @@ function SignInPageInner({credentials, account, activeClients = [], preferredLoc
     let welcomeText = '';
     const headerText = translate('login.hero.header');
 
+    const userLogin = Str.removeSMSDomain(credentials?.login ?? '');
+
+    // replacing spaces with "hard spaces" to prevent breaking the number
```

Comment:
NAB: You don't need this comment (or surrounding whitespace) since this change is occurring within `formatPhoneNumber` now